tabbycat/importer/templates/archive_importer.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block head-title %}{% endblock %}
+{% block page-title %}{% trans "Import tournament archive" context "page title" %}{% endblock %}
+{% block nav %}{% endblock %}
```

Comment:
Wants to get the tournament slug, even though the nav is empty (similarly to `test-email.html`)

Reply:
Resolved. Had to set `user_groups` in the view.